aurora/src/test/java/com/market/banica/aurora/AuroraApplicationTest.java

```diff
@@ -0,0 +1,15 @@
+package com.market.banica.aurora;
+
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.annotation.DirtiesContext;
+
+@SpringBootTest
+@DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_CLASS)
+public class AuroraApplicationTest {
```

Comment:
Why would we need this empty class?

Reply:
This is the default spring boot test, it checks if the context loads successfully. It was missing so i added it manually.